nucypher/characters/lawful.py

```diff
@@ -296,7 +296,7 @@ def grant(self,
         # value and expiration combinations on a limited number of Ursulas;
         # Users may decide to inject some market strategies here.
         #
-        # TODO: #289
+        # TODO: 289
```

Comment:
I think @KPrasch and you have been fighting a silent war: he puts the number sign in TODOs and you remove it.

Reply:
https://getyarn.io/yarn-clip/b855c713-3d55-45c6-8c58-feb088baba55